packages/components/src/Menu/Menu.test.tsx

```diff
@@ -139,6 +139,106 @@ describe("Menu", () => {
     await userEvent.click(getByRole("button"));
     expect(clickHandler).toHaveBeenCalledTimes(1);
   });
+
+  describe("icon colors", () => {
+    it("should default to --color-icon", async () => {
+      render(
+        <Menu
+          activator={<Button label="Menu" />}
+          items={[
+            {
+              actions: [
+                {
+                  label: "Down",
+                  icon: "arrowDown",
+                },
+              ],
+            },
+          ]}
+        />,
+      );
+
+      await userEvent.click(screen.getByRole("button"));
+
+      const iconSvg = screen.getByTestId("arrowDown");
+      expect(iconSvg.style.fill).toBe("var(--color-icon)");
+    });
+
+    it("should respect special icon colors", async () => {
+      render(
+        <Menu
+          activator={<Button label="Menu" />}
+          items={[
+            {
+              actions: [
+                {
+                  label: "New job",
+                  icon: "job",
+                },
+              ],
+            },
+          ]}
+        />,
+      );
+
+      await userEvent.click(screen.getByRole("button"));
+
+      const iconSvg = screen.getByTestId("job");
+      const pathElement = iconSvg.querySelectorAll("path")[0];
+      expect(pathElement.style.fill).toBe("var(--color-job)");
+    });
+
+    it("should allow overriding icon colors", async () => {
+      render(
+        <Menu
+          activator={<Button label="Menu" />}
+          items={[
+            {
+              actions: [
+                {
+                  label: "New job",
+                  icon: "job",
+                  iconColor: "icon",
+                },
+              ],
+            },
+          ]}
+        />,
+      );
+
+      await userEvent.click(screen.getByRole("button"));
+
+      const iconSvg = screen.getByTestId("job");
+      const pathElement = iconSvg.querySelectorAll("path")[0];
+      expect(pathElement.style.fill).toBe("var(--color-icon)");
```

Comment:
That's not a huge deal, but, perhaps, the following assertion would be a bit cleaner?

```suggestion
      const iconSvg = screen.getByTestId("job");
      const pathElement = iconSvg.querySelector("path");
      expect(pathElement).toHaveStyle("fill: var(--color-icon)");
```